backend/src/controllers/booking/currentBookingsController.ts

```diff
@@ -125,7 +125,10 @@ export const filterCurrentBookings = (
                 return false;
             }
 
-            const now = DateTime.now().toUTC().toISO();
+            const now = DateTime.now()
```

Comment:
Tää menee näin varmasti nyt ihan hyvin, mutta mietin että pitäisikö jossakin vaiheessa silti tehdä se muutos siihen, että backend pyörii UTC ajassa eli +00 ajassa🤔 Nythän jos backend pyörisi esim jenkeissä niin se olisi silti suomen ajassa, joka nyt ei kuitenkaan ole minkään sortin standardi.

Mutta tällein korjauksena mun puolesta OK, jos vaan toimii!

Reply:
Joo se on kyllä ihan totta että olisi varmaan paras silti tehdä backendiin se muutos että pyörii UTC ajassa koko backend, niin olis kaikkialla backendissa samalla tavalla sitten. Ja juu tää lähinnä temporary fix.